modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+----
+
+.. For Ansible- and Helm-based Operator projects, add a `help` target to the `Makefile`.
++
+Ansible- and Helm-based projects now provide `help` target in the `Makefile` by default, similar to a `--help` flag. You can manually add this target to your `Makefile` using the following lines:
++
+.`help` target
+[%collapsible]
+====
+[source,make]
+----
+##@ General
+
+# The help target prints out all targets with their descriptions organized
+# beneath their categories. The categories are represented by '##@' and the
+# target descriptions by '##'. The awk commands is responsible for reading the
+# entire set of makefiles included in this invocation, looking for lines of the
+# file as xyz: ## something, and then pretty-format the target and help. Then,
+# if there's a line with ##@ something, that gets pretty-printed as a category.
+# More info on the usage of ANSI control characters for terminal formatting:
+# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
+# More info on the awk command:
+# http://linuxcommand.org/lc3_adv_awk.php
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+----
+====
+
+.. Add `opm` and `catalog-build` targets, which enable you to create your own catalogs for your Operator or add your Operator bundles to an existing catalog:
+
+... Add the targets to your `Makefile` using the following lines:
++
+.`opm` and `catalog-build` targets
+[%collapsible]
+====
+[source,make]
+----
+.PHONY: opm
+OPM = ./bin/opm
+opm:
+ifeq (,$(wildcard $(OPM)))
+ifeq (,$(shell which opm 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPM)) ;\
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$(OS)-$(ARCH)-opm ;\
+	chmod +x $(OPM) ;\
+	}
+else
+OPM = $(shell which opm)
+endif
+endif
+BUNDLE_IMGS ?= $(BUNDLE_IMG)
+CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION) ifneq ($(origin CATALOG_BASE_IMG), undefined) FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG) endif
+.PHONY: catalog-build
+catalog-build: opm
+	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+
+.PHONY: catalog-push
+catalog-push: ## Push the catalog image.
+	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+----
+====
+
+... If you are updating a Go-based Operator project, also add the following `Makefile` variables:
++
+.`Makefile` variables
+[%collapsible]
+====
+[source,make]
+----
+OS = $(shell go env GOOS)
+ARCH = $(shell go env GOARCH)
+----
+====
+
+.. For Go-based Operator projects, set the `SHELL` variable in your `Makefile` to the system `bash` binary.
++
+Importing the `setup-envtest.sh` script requires bash, so the `SHELL` variable must be set to `bash` with error options:
```

Comment:
If that first bash should not be in backticks and you find yourself with spare time, please share with me.  I thought any instance of an OS command required backticks.

Reply:
Yeah, that one should either be `bash` or I suppose if it was a software-name reference instead, Bash (no markup) seems correct: https://www.gnu.org/software/bash/